### ISSUE TITLE
Improve links selection at various place

### ISSUE THREAD

**Issue by dcadiou** [grp-attestoodle/moodle-tool_attestoodle#160]
Many locations in the plugin contains links that do not target the best place to quickly navigate from one point to another.

Few examples : 
- in Site Administration / Course / Attestoodle / list of trainings leads to category instead of directly targeting training management page (or training view page as suggested in #159 ). 
- All attestoodle links to trainings should leads to training view or management, not to category. Links to category should be explicitly labelled as so
- (to be completed...)

**Comment by Hipjea** [comment]
I agree with this, the plugin needs a better interface overall. Moreover, its access isn't intuitive.
I was genuinely searching for the plugin settings under `Administration / Reports` instead of `Administration / Courses`.
